src/libos/crates/async-io/src/file/file.rs

```diff
@@ -69,6 +65,7 @@ pub trait File: Debug + Sync + Send {
         StatusFlags::empty()
     }
 
+    #[allow(unused_variables)]
```

Comment:
Should we use the same fix as others?
fn set_status_flags(&self, _new_status: StatusFlags) -> Result<()>

Reply:
Yes, will update